File: 프로그래머스/lv0/120818. 옷가게 할인 받기/옷가게 할인 받기.py
import math

def solution(price):
    answer = 0
    if price >= 100000 and price < 300000 :
        answer = price * 0.95
    elif price >= 300000 and price < 500000:
        answer = price * 0.9
    elif price >= 500000 :
        answer = price * 0.8
    else :
        answer = price
    return math.floor(answer)

File: 프로그래머스/unrated/120818. 옷가게 할인 받기/옷가게 할인 받기.java
class Solution {
    public int solution(int price) {
        
        if(price >= 500_000) price *= 0.8;
        else if(price >= 300_000) price *= 0.9;
        else if(price >= 100_000) price *= 0.95;
        
        return (int)price;
    }
}